projects/daily_dribble_newsletter/index.html

```diff
@@ -3,24 +3,26 @@
 <html>
   <head>
     <link rel="stylesheet" href="styles.css" />
+    <style>
+      
+    </style>
   </head>
   <body>
-    <h1>THE DAILY DRIBBLE</h1>
+    <div class="container">
+      <h1>THE DAILY DRIBBLE</h1>
 
-    <!-- Image credit: Thomas Serer https://unsplash.com/@jesusance -->
-    <img src="images/stadium.jpg" />
+      <!-- Image credit: Thomas Serer https://unsplash.com/@jesusance -->
+      <img src="images/stadium.jpg" /><br><br>
+      <strong><em>Your daily dose of soccer stories from on and off the pitch. Interviews, analyses, breaking news and more!</em></strong>
```

Comment:
These two should also have the same width. Currently, the text stretches beyond the image width (note that the text also takes two lines in the proposed design).